### PR TITLE
Update nl_NL.json

### PR DESCRIPTION
**Pull request by JackdeHeer**
I fixed a couple of small things I've spotted this evening. Will check some more in the next days. :)  

We still mix up some stuff (probably because of the different translators): 
English: 'You - yours'
Dutch - formal: 'u - uw'
Dutch - informal: 'je, jij, jou - je, jouw'

Do you guys prefer the formal or informal way when addressing somebody? 
The bigger Dutch banks *ahum* tend to go the informal way. 

Don't forget things like: 'alstublieft, alsjeblieft' as well.

### CODE REVIEW

File: locales/nl_NL.json [hunk 1]
```diff
@@ -192,7 +192,8 @@
   "top-nav": {
     "exchange-status": {
       "operational": "Exchange volledig operationeel",
-      "maintenance": "Exchange in onderhoud"
+      "maintenance": "Exchange in onder
```
Comment (JackdeHeer) [inline locales/nl_NL.json:195]: Sloppy mistake...

**Comment by NichaT** [comment]
@JackdeHeer I agree there should be consistency between formal and informal.

I personally went with formal when translating. Although I think that informal could be done as well. The third option could be something in between, when using the exchange, informal and when making an account, submitting KYC , it could be done in an informal way.

**Comment by lexipenia** [comment]
@JackdeHeer @NichaT Could you please tell me when the changes on this branch are ready to merge?

**Comment by lexipenia** [comment]
@JackdeHeer @NichaT Should I merge this in now?

**Comment by NichaT** [comment]
@lexipenia yes 👍🏼.

**Comment by JackdeHeer** [comment]
Sure thing!

Op ma 17 aug. 2020 11:52 schreef Nicha Zwarenstein Tutunji <
notifications@github.com>:

**Pull request merged**